### PR TITLE
Added .error.cf possibility to acceptance tests

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4718/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4718/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4718/

**Comment by olehermanse** [comment]
@cf-bottom please try this in jenkins, with exotics

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4721/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4721/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4721/

**Comment by olehermanse** [comment]
RHEL 5 only, now using `egrep`:

https://ci.cfengine.com/job/pr-pipeline/4722/

[Build Status](https://ci.cfengine.com/job/pr-pipeline/4722/)

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4723/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4723/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4723/

**Comment by olehermanse** [comment]
Redhat 5 only using `fgrep`:

https://ci.cfengine.com/job/pr-pipeline/4724/

[Build Status](https://ci.cfengine.com/job/pr-pipeline/4723/)

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4725/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4725/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4725/

File: tests/acceptance/testall [hunk 1]
```diff
@@ -330,6 +333,10 @@ runtest() {
     then
         SKIP=1
         SKIPREASON="${COLOR_WARNING}'errorexit' tests are disabled${COLOR_NORMAL}"
+    elif [ $ERRORLOG_TESTS = 0  -a  $TEST_TYPE = errorlogt ]
```
Comment (vpodzime) [inline tests/acceptance/testall]: `errorlogt`?

File: tests/acceptance/testall [hunk 2]
```diff
@@ -516,9 +523,36 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP CFENGINE_TEST_OVERRIDE_EXTENSION_LIBR
...
+                RESULT_MSG="${COLOR_FAILURE}FAIL (Failed to exit with error!)${COLOR_NORMAL}"
+            fi
+        elif [ $TEST_TYPE = errorlog ]
         then
+            if [ $RETVAL != 0 ]
+            then
+                RESULT=FAIL
+                RESULT_MSG="${COLOR_FAILURE}FAIL (NON-ZERO EXIT CODE!)${COLOR_NORMAL}"
+            elif fgrep 'error: ' "$OUTFILE" > /dev/null
+            # Regex: start of line, timestamp without spaces, spaces, error:, space
```
Comment (vpodzime) [inline tests/acceptance/testall]: Obsolete comment.

File: tests/acceptance/testall (hunk 1 above)
Comment (olehermanse) [reply tests/acceptance/testall]: Thanks, fixed!

File: tests/acceptance/testall (hunk 2 above)
Comment (olehermanse) [reply tests/acceptance/testall]: Thanks! Removed.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4730/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4730/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4730/

**Comment by olehermanse** [comment]
Cherry-pick: https://github.com/cfengine/core/pull/4098

**Pull request merged**